features/app/components/native/App.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import { Navigator } from 'react-native';
+import { Provider } from 'react-redux';
+
+import { WelcomePage } from '../../../welcome';
+
+/**
+ * Root application component.
+ *
+ * @extends Component
+ */
+export class App extends Component {
+    /**
+     * Initializes a new Root instance.
+     *
+     * @param {Object} props - The read-only properties with which the new
+     *      instance is to be initialized.
+     */
+    constructor(props) {
+        super(props);
+
+        // Bind event handlers so they are only bound once for every instance.
+        this._navigatorRenderScene = this._navigatorRenderScene.bind(this);
+    }
+
+    /**
+     * Implements React's {@link Component#render()}.
+     *
+     * @inheritdoc
+     * @returns {ReactElement}
+     */
+    render() {
+        return (
+            <Provider store={this.props.store}>
+                <Navigator
+                    initialRoute={{
+                        title: 'Jitsi Meet',
+                        component: WelcomePage
```

Comment:
Please comply with the coding style which requires sorting by alphabetical order!